src/shared/components/mutationTable/IMutationTableProps.ts

```diff
@@ -9,6 +9,7 @@ export interface IMutationTableProps extends IEnhancedReactTableProps<MutationTa
 {
     title?:string;
     samples?: ClinicalDataBySampleId;
+    onVisibleRowsChange?:(data:MutationTableRowData[]) => any;
```

Comment:
It doesn't have to be void though, if you wanted to use the same function for another purpose or something this lets you do that.